Author: Andreas Beckmann <anbe@debian.org>
Description: link with -lboost_date_time
 needed when building without optimization
 will be omitted by --as-needed in other cases

--- a/unix/configure.ac
+++ b/unix/configure.ac
@@ -310,7 +310,7 @@ fi
 AC_MSG_CHECKING([whether the boost thread library is usable])
 SAVED_LIBS=$LIBS
 boost_thread_links=0
-for extralib in '' '-lboost_system'
+for extralib in '' '-lboost_system -lboost_date_time'
 do
   LIBS=$SAVED_LIBS
   LIBS="$LIBS $extralib"
